### DISCUSSION TITLE
is there an EXP function to return the exponential value of an expression

### DISCUSSION THREAD

**Discussion by phrrngtn** [duckdb/duckdb#8263] [Q&A]
I see ExpOperator in  src/core_functions/scalar/math/numeric.cpp and it looks like the right thing but I don't see it referred to anywhere else.

I was expecting to see it in: Function Compatibility with Other Systems #193

`ln()` exists but I don't see an `e()`

workaround is to create a macro: `create macro exp(n) AS 2.718281828459045**n;`

**Comment by l1t1** [comment]
yes
```
D select exp(1) a ;
┌───────────────────┐
│         a         │
│      double       │
├───────────────────┤
│ 2.718281828459045 │
└───────────────────┘
```

**Comment by phrrngtn** [comment]
thanks! I did not see any reference to it in the documentation and -- somewhat bizarrely -- it seems I did not try the obvious thing. Sorry about that! To atone for my inattention, I will try and submit a PR for the docs.

**Comment by phrrngtn** [comment]
https://github.com/duckdb/duckdb-web/pull/882
